packages/react-native/src/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { HelloWorld } from '..';
+import TestRenderer from 'react-test-renderer';
```

Comment:
Curious, can you use react-testing-library in React Native?

Reply:
[RN Testing Library](https://testing-library.com/docs/react-native-testing-library/intro/)

We have `"@testing-library/react-hooks"` in the deps already